tests/MongoDB.Driver.Tests/Specifications/crud/CrudTestRunner.cs

```diff
@@ -65,7 +62,7 @@ public void RunTestDefinition(BsonDocument definition, BsonDocument test)
 
             var databaseName = GetDatabaseName(definition);
             var collectionName = GetCollectionName(definition);
-            DropCollection(databaseName, collectionName);
+            RemoveCollectionData(databaseName, collectionName);
```

Comment:
Sometimes just removing the data isn't enough.

The collection might have indexes that cause subsequent tests to fail.

We need to drop the collection.

If the problem is that we're dropping a collection in the admin database then we should be using a different database for testing.

Reply:
I know it, but I thought maybe it's not the case for CRUD. If not so, I guess the best option is updating json tests, because I don't think that replacing admin name on something else here is good enough, because we will need to ensure that some other test steps won't expect the admin value in their logic

Reply:
I didn't think about the fact that this was local to this class and only affected these tests.

Since we know it works with the CRUD JSON test files this seems OK.

Reply:
Can you explain why we need to make this change.  Is the value of `DriverTestConfiguration.DatabaseNamespace.DatabaseName` equal to `admin`?  I don't think it is.  And there is only one CRUD test where the database_name equals admin, and that test doesn't have a collection name, so it shouldn't have to drop any collection (I suspect for that test this is just picking up the default collection name.

Reply:
you're probably right, I didn't check the CRUD logic, I just saw the thrown error. The reason why I did this change is `db-aggregate.json`, but as you noticed this test doesn't have the `collection_name` node at all, so maybe it will be enough just to update `Drop` logic to call it only if we have a collection name in the json file. Will do it in the next commit.

Reply:
Discussed offline. It's ok to have the current version since it's the simplest way.